colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+    array([422, 512, 512])
+    """
+    intOut = (isinstance(outRange[0], int) and
+              isinstance(outRange[1], int) and
+              isinstance(outRange[2], int) and
+              isinstance(outRange[3], int))
+    r, g, b = tsplit(rgb)
+    Yc = 0.2627*r + 0.6780*g + 0.0593*b
+    Yc = REC_2020_COLOURSPACE.OECF(Yc, is_10_bits_system=is_10_bits_system)
+    r = REC_2020_COLOURSPACE.OECF(r, is_10_bits_system=is_10_bits_system)
+    b = REC_2020_COLOURSPACE.OECF(b, is_10_bits_system=is_10_bits_system)
+    Cbc = np.where((b - Yc) <= 0.0, (b - Yc)/1.9404, (b - Yc)/1.5816)
+    Crc = np.where((r - Yc) <= 0.0, (r - Yc)/1.7184, (r - Yc)/0.9936)
+    Yc *= outRange[1] - outRange[0]
+    Yc += outRange[0]
+    Cbc *= outRange[3] - outRange[2]
+    Crc *= outRange[3] - outRange[2]
+    Cbc += (outRange[3] + outRange[2])/2
+    Crc += (outRange[3] + outRange[2])/2
+    if intOut:
+        Yc = np.round(Yc).astype(int)
+        Cbc = np.round(Cbc).astype(int)
+        Crc = np.round(Crc).astype(int)
+    YcCbcCrc = tstack((Yc, Cbc, Crc))
+    return YcCbcCrc
+
+
+def YcCbcCrc_to_rgb(YcCbcCrc, inRange=RANGE['legal_10_YC'],
+                    is_10_bits_system=True):
+    """
+    Converts an array of Yc'Cbc'Crc' values to the corresponding RGB (linear)
+    values.
+
+    Note: This fuction is specifically for use with ITU-R BT.2020 (Rec.2020)
+    when using the constant luminance implementation.
+    See https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+    R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+
+    Parameters
+    ----------
+    YcCbcCrc : array_like
+        Input Yc'Cbc'Crc' values. These may be floats or integers.
+    inRange : array of four values (optional)
+        The values of Yc' corresponding to 0 IRE and 100 IRE,
+        and the minimum and maximum values of Cbc' and Crc'.
+        These may be floats or integer code values.
+        Default = RANGE['legal_10_YC']
+    is_10_bits_system : bool (optional)
+        The Rec.2020 EOCF varies slightly for 10 and 12 bit systems.
+        Default: True
+
+
+    Returns
+    -------
+    ndarray
+        RGB values corresponding to the input colour.
+        These will be floating point linear values.
+
+    Examples
+    --------
+    >>> YcCbcCrc = np.array([1689, 2048, 2048])
+    >>> YcCbcCrc_to_rgb(YcCbcCrc,
+    ...                 inRange=RANGE['legal_12_YC_int'],
+    ...                 is_10_bits_system=False)  # doctest: +ELLIPSIS
+    array([ 0.18009037,  0.18009037,  0.18009037])
+    """
+    Yc, Cbc, Crc = tsplit(YcCbcCrc.astype(float))
+    Yc -= inRange[0]
+    Cbc -= (inRange[3] + inRange[2])/2.0
+    Crc -= (inRange[3] + inRange[2])/2.0
+    Yc *= 1.0/(inRange[1] - inRange[0])
+    Cbc *= 1.0/(inRange[3] - inRange[2])
+    Crc *= 1.0/(inRange[3] - inRange[2])
+    b = np.where(Cbc <= 0.0, Cbc*1.9404 + Yc, Cbc*1.5816 + Yc)
+    r = np.where(Crc <= 0.0, Crc*1.7184 + Yc, Crc*0.9936 + Yc)
+    Yc = REC_2020_COLOURSPACE.EOCF(Yc, is_10_bits_system=is_10_bits_system)
```

Comment:
Yeah I would leave it as is for now, I'm trying to get the [cctf branch](https://github.com/colour-science/colour/blob/feature/cctf/) wrapped so that you can rebase your work on top of develop when it gets merged. I think I'm not too far on finishing on that one, maybe a PR this weekend.